### DISCUSSION TITLE
Change the user Rich Presence (Now Playing, Streaming, ...)

### DISCUSSION THREAD

**Discussion by DeadLyBro** [rigwild/discord-self-bot-console#37] [Ideas]
Hi i want to try make Custom Activity with API. 

Can we do Custom Activity?  

Its like "Playing", "Listening", "Streaming" or "Custom" (with emoji). I read Discord API a little bit, but idk how to do.

**Comment by DeadLyBro** [comment]
Can we do Custom Activity? @rigwild 
I think we should use Gateway. ( Its writing on API. But idk how to do. )

**Comment by rigwild** [comment]
This is out of scope of this project.

See Discord documentation:
- [Introducing Rich Presence](https://discord.com/developers/docs/rich-presence/how-to)
- [SDK Starter Guide](https://discord.com/developers/docs/game-sdk/sdk-starter-guide)

You can check those:
- https://github.com/maximmax42/Discord-CustomRP
- https://github.com/ThatOneCalculator/DiscordRPCMaker

**Reply by DeadLyBro** [reply]
Thank for answer.
